atspi/examples/selected-text.rs

```diff
@@ -0,0 +1,36 @@
+//! This example demonstrates how to get the currently selected text.
+//!
+//! ```sh
+//! cargo run --example selected-text
+//! ```
+//! Authors:
+//!    Colton Loftus
+
+use atspi::{events::object::{TextSelectionChangedEvent}, ObjectEvents};
+use atspi_proxies::{accessible::ObjectRefExt, proxy_ext::ProxyExt};
+use futures_lite::stream::StreamExt;
+use std::{error::Error};
+
+#[async_std::main]
+async fn main() -> Result<(), Box<dyn Error>> {
+	let atspi = atspi::AccessibilityConnection::new().await?;
+    let conn = atspi.connection();
+	atspi.register_event::<ObjectEvents>().await?;
+
+	let mut events = atspi.event_stream();
+
+	while let Some(ev) = events.next().await {
+        match ev {
+            Ok(ev) => {
+                if let Ok(ev) = <TextSelectionChangedEvent>::try_from(ev) {
+                    let text_proxy = ev.item.into_accessible_proxy(conn).await?.proxies().await?.text().await?;
+                    const ASSUME_ONLY_ONE_SELECTED_RANGE: i32 = 0;
+                    let (start, end) = text_proxy.get_selection(ASSUME_ONLY_ONE_SELECTED_RANGE).await?;
```

Comment:
Where did you find the meaning of the argument? Just curious. If it is a credible source, let's document it on our method.

Reply:
Where does it say that 0 indicates 'only one selection'?

Reply:
So it is an index?